espnet2/fileio/score_scp.py

```diff
@@ -83,12 +83,20 @@ def __getitem__(self, key):
                     else:
                         notes_list.append(NOTE("P", 0, st, st + dur))
                     prepitch = 0
+                    st += dur
+                    continue
                 else:  # normal note for one syllable
                     notes_list.append(NOTE(lr, note.pitch.midi, st, st + dur))
                 prepitch = note.pitch.midi
-                for arti in note.articulations:  # <br> is tagged as a notation
-                    if arti.name in ["breath mark"]:  # up-bow?
-                        notes_list.append(NOTE("B", 0, st, st))  # , 0))
+                for arti in note.articulations:
+                    # NOTE(Yuning): By default, 'breath mark' appears at the end of
```

Comment:
I think this is worth noting. Can you also add this info to the doc as well

Reply:
Fixed.